src/sections/z_other/Footer-classic/index.js

```diff
@@ -1,14 +1,14 @@
 import React from "react";
 import { Link } from "gatsby";
 
-import {
-  FaFacebookF,
-  FaTwitter,
-  FaLinkedinIn,
-  FaEnvelope,
-  FaPinterestP,
-  FaInstagram
-} from "react-icons/fa";
+// import {
```

Comment:
Since you're commenting these, do we need it at all?

Reply:
Reason being these are complete components/sections and removing just those icons will trouble later on if we use this component.
Though, I doubt we will ever use it, so just commented out.